build/build.proj

```diff
@@ -88,7 +80,7 @@
 
   </Target>
 
-  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;BuildModernVsixPackages;Test" />
-  <Target Name="Rebuild" DependsOnTargets="RestorePackages;RebuildSolution;RebuildModernVsixPackages;Test" />
+  <Target Name="Build" DependsOnTargets="RestorePackages;BuildSolution;Test" />
```

Comment:
Jenkins will anyway not build modern VSIXes as that target is conditioned on running in microbuild.